api/views/personal_info.py

```diff
@@ -56,6 +56,7 @@ def get_serializer_class(self) -> type[BaseSerializer[PersonalInfo]]:
     @extend_schema(
         parameters=[
             OpenApiParameter(name="registration_telegram_bot_chat_id", type=int, required=True),
+            OpenApiParameter(name="email", type=str, required=False),
```

Comment:
```suggestion
```

Please revert this change, this is the wrong place for this parameter. It is a view for the bot that checks the existence of chat ID :)